src/Core/ConfigurationParser.rei

```diff
@@ -9,3 +9,5 @@ let ofJson: Yojson.Safe.t => result(Configuration.t, string);
 let ofString: string => result(Configuration.t, string);
 
 let ofFile: string => result(Configuration.t, string);
+
+let parseStringList: [> | `List(list([> | `String('a)])) | `String('a)] => list('a);
```

Comment:
Could this be tested without exposing internals? Seems like every configuration item is optional, so you could just pass in `experimental.viml` or `files.exclude` and then check the final parsed value, couldn't you?

Reply:
You're right. tested at a higher level.